File: Fac.py
def factorial(num):
    fac = 1
    for i in range(1, num+1):
        fac *= i
    return fac

if __name__ == "__main__":
    num = int(input("Enter a number : "))
    fac = factorial(num)
    print("Factorial of the number is", fac)

File: Fac.js
let factorial = (num) => {
    let fac = 1;
    for (let i = 1; i != num + 1; i++) {
        fac *= i;
    }
    return fac;
}

let num = 5;
let fac = factorial(num);
console.log("The factorial of the number is " + fac);
